src/api.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+
+const newsClient = axios.create({
+    baseURL: "https://nc-news-api-ne3e.onrender.com/api"
+}) 
+
+const getArticles = () => {
+    return newsClient.get("/articles")
+    .then((response) => {
+        return response
+    })
+    .catch((err) => {
+        console.log(err)
+    })
```

Comment:
A .catch(...) shouldnt be on this level. We should allow this function to fail if it does and have a .catch in the component that is using this function.